authlib/integrations/base_client/remote_app.py

```diff
@@ -197,5 +197,5 @@ def load_key(header, payload):
             claims_options=claims_options,
             claims_params=claims_params,
         )
-        claims.validate(leeway=120)
+        claims.validate(leeway)
```

Comment:
```suggestion
        claims.validate(leeway=leeway)
```

Because the first value of `validate` is `now`.

Reply:
Good catch. Thanks.